multicore_runtime/static_files.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Static file serving functionality invoked by wsgi_config.py."""
+
+import logging
+import mimetypes
+import os
+import re
+
+from werkzeug.wrappers import Request
+from werkzeug.wrappers import Response
+from werkzeug.wsgi import wrap_file
+
+
+def static_app_for_regex_and_files(regex, files, upload, mime_type=None):
+  """Returns a WSGI app that serves static files.
+
+  Args:
+    regex: A url-matching regex as specified in appinfo.URLMap.
+    files: A static_files definition as specified in appinfo.URLMap.
```

Comment:
Not sure I like the name `files` here. I'd use words like `mapping` or `template` etc. since this is essentially a template to map the URL into a filesystem path, right?